src/ray/gcs/redis_context.h

```diff
@@ -113,6 +122,46 @@ class RedisContext {
   redisAsyncContext *subscribe_context_;
 };
 
+template <typename ID>
+Status RedisContext::RunAsync(const std::string &command, const ID &id,
```

Comment:
why move this function from `.cc` to `.h`?

Reply:
Before this PR, all ID's base class is `UniqueID` which has 20-byte data. This function is changed to a template function, so it is moved to the header file.